index/tests.py

```diff
@@ -1,23 +1,170 @@
 import unittest
```

Comment:
I think we should change this to `from django.test import TestCase`. I included this in the initial commit of this file, so I'm sorry about that. See the [docs](https://docs.djangoproject.com/en/2.1/topics/testing/overview/#writing-tests) for details.